### PR TITLE
Add templates for github issues & pull requests

### PR DESCRIPTION
**Pull request by bear454**
re: #1763
based on: #1764

### CODE REVIEW

**Comment by differentreality** [comment]
I would **not** make the templates mandatory, but if you think they are helpful to new contributors, I don't object in having them.

@hennevogel @lagartoflojo @ChrisBr @Ana06 what do you think?

**Comment by lagartoflojo** [comment]
I think that these templates are not supposed to be useful for new contributors, but rather for existing contributors, who want to receive quality issue reports, feature requests and pull requests. In that regard, I think this is a good starting point, and if we see that people don't use the templates correctly, we can very easy adjust them in the future. 👍

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
++**I'm submitting a ..**
```
Comment (lagartoflojo) [inline .github/ISSUE_TEMPLATE.md]: Are these `++` at the beginning of each line necessary?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
++- [ ] Bug Report
++- [ ] Feature Request
++
++**Current behavior:**
++<!-- Describe the bug and how it creeped in. -->
```
Comment (lagartoflojo) [inline .github/ISSUE_TEMPLATE.md]: If you asked me "how it creeped in," I would try to look for the specific commit that changed the line the triggered the bug, and I don't think that's what we expect here. I'd just leave it at "Describe the bug."

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
++
++**Expected behavior:**
```
Comment (lagartoflojo) [inline .github/ISSUE_TEMPLATE.md]: Just to be more explicit, I'd say "Expected correct behavior"

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
++<!-- Describe what the behavior would be without the bug. -->
```
Comment (lagartoflojo) [inline .github/ISSUE_TEMPLATE.md]: How about:

> Describe what should happen without the bug

File: .github/ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
++
++**Steps to reproduce:**
++<!-- If you are able to illustrate the bug or feature request with an example, please provide steps to reproduce -->
```
Comment (lagartoflojo) [inline .github/ISSUE_TEMPLATE.md]: reproduce **it**

File: .github/ISSUE_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
++
++**Related code:**
```
Comment (lagartoflojo) [inline .github/ISSUE_TEMPLATE.md]: Not sure if we need this section. Bugs will not necessarily come from developers.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
++#### Checklist
++
++- [ ] I have read the [Contribution & Best practices Guide](https://github.com/openSUSE/osem/blob/master/CONTRIBUTING.md).
++- [ ] My branch is up-to-date with the Upstream `master` branch.
++- [ ] The tests pass locally with my changes.
++- [ ] I have added tests that prove my fix is effective or that my feature works(if appropriate).
```
Comment (lagartoflojo) [inline .github/PULL_REQUEST_TEMPLATE.md]: missing space between `works(if`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
++#### Short description of what this resolves:
++
++
++#### Changes proposed in this pull request:
++
++-
++-
++-
++
++#### Fixes Issue
```
Comment (lagartoflojo) [inline .github/PULL_REQUEST_TEMPLATE.md]: The ISSUE_TEMPLATE.md uses `**asterisks**` for each section, but in this file we use `#### four hashes`. I would normalize and use the same one in both files (I don't have any preference *which*).

**Comment by bear454** [comment]
Thanks for the corrections @lagartoflojo !

**Comment by bear454** [comment]
@lagartoflojo your review got dismissed when I rebased onto master. Would you mind checking the box one more time?

**Pull request merged**